src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -91,9 +99,9 @@
    public void initializeUI(UIBuilder builder) throws Exception
    {
       UIContext context = builder.getUIContext();
+      Mission mission = (Mission) context.getAttributeMap().get(Mission.class);
```

Comment:
This needs to be inside the anonymous class, otherwise it won't get the updated value properly